setTRIMDAC_and_Mask.py

```diff
@@ -77,3 +76,70 @@
         mask_yes = int(cc[2])
         setChannelRegister(testSuite.glib, options.gtx, port, channel, mask_yes, 0x0, trimDAC, debug = False)
         pass
+    g.close()
+    pass
+trimDACfileList.close()
+
+if (options.do_thresh):
+    THRESH_ABS = 0.1
+    THRESH_REL = 0.05
+    THRESH_MAX = 250
+    THRESH_MIN = 0
+    N_EVENTS = 1000.00
+
+    configureScanModule(testSuite.glib, options.gtx, 0, 0, scanmin = THRESH_MIN, scanmax = THRESH_MAX, numtrigs = int(N_EVENTS), useUltra = True, debug = options.debug)
+    printScanConfiguration(testSuite.glib, options.gtx, useUltra = True, debug = options.debug)
+    startScanModule(testSuite.glib, options.gtx, useUltra = True, debug = options.debug)
+    UltraResults = getUltraScanResults(testSuite.glib, options.gtx, THRESH_MAX - THRESH_MIN + 1, options.debug)
+
+    if options.debug:
+        raw_input ("Press Enter to Continue")
+    print
+    print "Starting Preliminary Threshold Scan"
+    print
+    for n in testSuite.presentVFAT2sSingle:
+        # f = open("%s_Data_GLIB_IP_%s_VFAT2_%d_ID_0x%04x"%(str(Date),str(options.slot),n,testSuite.chipIDs[n]&0xffff),'w')
```

Comment:
Is the writing out of these types of files really necessary for this script?
If so, the naming convention needs to be the same as in the updated code, and in fact, the whole block should be written as a function called here, rather than copied and pasted here